tests/Controller/BlogControllerTest.php

```diff
@@ -36,6 +36,8 @@ public function testIndex()
             $crawler->filter('article.post'),
             'The homepage displays the right number of posts.'
         );
+        
+        $this->assertEquals(200, $client->getResponse()->getStatusCode());
```

Comment:
Hello @DurandSacha, can you use the following code instead : 
`$this->assertResponseIsSuccessful()`

Reply:
thank you dear friend